manifest

```diff
@@ -11,13 +11,22 @@ export DOCUMENTATION_URL="https://chimeraos.org/about"
 export BUG_REPORT_URL="https://github.com/ChimeraOS/chimeraos/issues"
 
 export PACKAGES="\
-	gnome-desktop \
 	gnome-control-center \
 	gnome-session \
 	gnome-shell \
 	gnome-software \
 	gnome-terminal \
+	gnome-keyring \
+	gnome-menus \
+	gnome-system-monitor \
 	nautilus \
+	sushi \
+	eog \
+	evince \
+	file-roller \
+	gedit \
+	xdg-user-dirs-gtk \
```

Comment:
These directories seem rather useless to me.

Reply:
They add to the desktop experience. Gnome and some flathub apps would rely on the default xdg directories to be there for downloads, images and documents. This only provides a mean for the desktop to manage them.